README.md

```diff
@@ -61,22 +61,16 @@ You can also override the following methods and properties:
 
 A function that is called once before the Scene is displayed. This will generally be where you configure your widget's dependencies.
 
-#### `environmentControlBuilders`
+#### `environmentControls`
 
-An optional list of `EnvironmentControlBuilder`s that allow you to add custom widgets to the Stager control panel. `EnvironmentControlBuilder` is a function similar to a `WidgetBuilder` – it accepts a `BuildContext` as a parameter and returns a `Widget`. It also accepts an additional `rebuildScene` callback that you can use to tell Stager to reconstruct your Scene.
-
-Stager calls these functions when constructing a Scene and places the widgets they return in the environment control panel.
+An optional list of `EnvironmentControl`s that allow you to add custom widgets to the Stager control panel. An `EnvironmentControl` provides a widget that allows the user to change values used when presenting a Scene. These values are persisted in a shared state, meaning controls can be reused between Scenes. Stager includes several of these controls that allow the user to toggle dark mode, change text scale, etc.
```

Comment:
Maybe say "stored" rather than "persisted" seeing as we don't really have persistence between runs yet?

Maybe say "state is preserved when the same controls are used in multiple scenes" rather than "controls can be reused between Scenes"?

Reply:
Good call, done.